src/email.rs

```diff
@@ -57,7 +57,7 @@ pub fn request(app: &AppConfig, email_addr: EmailAddress, client_id: &str, nonce
 
     // Generate the URL used to verify email address ownership.
     let href = format!("{}/confirm?session={}&code={}",
-                       app.base_url,
+                       app.public_url,
```

Comment:
All these config field name changes can then also go in a separate commit, I think, but again, many of these feel somewhat gratuitous.